src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+import "./Introduce.scss";
+import { Link, useNavigate } from "react-router-dom";
+
+
+function Introduce (){
+    const navigate = useNavigate();
+    
+    const handleButton = () => {
+        navigate('./product');
```

Comment:
```suggestion
        navigate('/product');
```
파일의 경로가 아닌 도메인의 경로를 작성해 주셔야 합니다!
Router 컴포넌트에서 지정한 경로를 넣는다고 생각하시면 쉽게 이해되실거 같아요!